### PR TITLE
PGD re-edit - group 9

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 1]
```diff
@@ -32,7 +32,7 @@ This is used by the upgrade scripts to transfer commit decisions
 saved in bdr.node_pre_commit catalog to the shared memory hash table.
 This will also be logged to the WAL and hence can be reloaded from
 WAL.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:34]: Unclear what the two cases of "this" mean. Can you suggest a revision?

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 2]
```diff
@@ -32,7 +32,7 @@ This is used by the upgrade scripts to transfer commit decisions
...
-
+<!-- Unclear what the two uses of "this" mean >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:35]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 3]
```diff
@@ -93,20 +93,19 @@ This is an internal function, mainly used by HARP.
...
 ### `bdr.decode_message_payload`
 
-PGD message payload function that decodes the consensus payloads to a more human-readable output.Used primarily by the `bdr.global_consensus_journal_details` debug view.
+PGD message payload function that decodes the consensus payloads to a more human-readable output. Used primarily by the `bdr.global_consensus_journal_details` debug view.
 
 ### `bdr.decode_message_response_payload`
 
 PGD message payload function that decodes the consensus payloads to a more human-readable output. Used primarily by the `bdr.global_consensus_journal_details` debug view.
-
+<!-- This description is identical to the one preceding >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:101]: This description is identical to the one preceding. Is there any differentiation?

Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:101]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 4]
```diff
@@ -223,21 +222,21 @@ Used internally to implement the `bdr.node_slots` view.
 ### `bdr.internal_alter_sequence_set_kind`
 
 PGD internal function used internally for replication of the various function calls.
-
+<! Could this just start with "Used internally" >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:225]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 5]
```diff
@@ -223,21 +222,21 @@ Used internally to implement the `bdr.node_slots` view.
...
 No longer used by the current version of PGD. Exists only for backward
 compatibility during rolling upgrades.
 
 ### `bdr.internal_replication_set_add_table`
 
 PGD internal function used internally for replication of the various function calls.
-
+<! Could this just start with "Used internally" >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:232]: Same query (Start with "used internally"?)

Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:232]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx [hunk 6]
```diff
@@ -223,21 +222,21 @@ Used internally to implement the `bdr.node_slots` view.
...
 No longer used by the current version of PGD. Exists only for backward
 compatibility during rolling upgrades.
 
 ### `bdr.internal_replication_set_remove_table`
 
 PGD internal function used internally for replication of the various function calls.
-
+<! Could this just start with "Used internally" >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:239]: Same query ("Used internally"?)

Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx:239]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/nodes.mdx [hunk 7]
```diff
@@ -17,7 +17,7 @@ indexdepth: 3
 -   `CATCHUP`: Initial sync phase is completed. Now the join is at the last step
     of retrieving and applying transactions that were performed on the upstream
     peer node since the join started.
--   `STANDBY`:  Node join finished, but hasn't yet started to broadcast changes.
+-   `STANDBY`:  Node join finished but hasn't yet started to broadcast changes.
     All joins spend some time in this state, but if defined as a logical standby,
     the node continues in this state.
 -   `PROMOTE`: Node was a logical standby and we just called `bdr.promote_node` to
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/nodes.mdx:23]: Wondering about the use of "we" here. Could this be changed to "you"?

File: product_docs/docs/pgd/5/reference/nodes.mdx [hunk 8]
```diff
@@ -34,26 +34,26 @@ indexdepth: 3
 -   `PART_CATCHUP`: Nonparting nodes synchronize any missing data from the
     recently parted node.
 -   `PARTED`: Node parting operation is now complete on all nodes.
-
+<!-- querying use of "we" >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/nodes.mdx:37]:
```suggestion
```

File: product_docs/docs/pgd/5/reference/functions-internal.mdx (hunk 2 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/functions-internal.mdx]:
```suggestion
```

**Comment by ebgitelman** [comment]
I think this is ready to go. Please approve.

File: product_docs/docs/pgd/5/reference/functions-internal.mdx (hunk 3 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/reference/functions-internal.mdx]: Yes, fixed.

File: product_docs/docs/pgd/5/reference/functions-internal.mdx (hunk 5 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/reference/functions-internal.mdx]: A function previously used internally....
(as we expand on in the next line)

File: product_docs/docs/pgd/5/reference/functions-internal.mdx (hunk 6 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/reference/functions-internal.mdx]: ditto

File: product_docs/docs/pgd/5/reference/nodes.mdx (hunk 7 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/reference/nodes.mdx]: Changed to "and `bdr.promote_node` was just called to...`

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**